client/client/modules/render/whole-genome-render/partials/scale/scaleRenderer.js

```diff
@@ -0,0 +1,169 @@
+import { drawingConfiguration } from '../../../core';
+import config from '../../whole-genome-config';
+
+export class ScaleRenderer {
+
+    constructor(container, canvasSize, drawingConfig, range){
+        this._ticksNumber;
+        this._ticks;
+        Object.assign(this, {
+            container,
+            canvasSize,
+            drawingConfig,
```

Comment:
The same as for the `ChromosomeColumnRenderer` - merge drawing config with the initial one, check if `canvasSize` is defined